content/blog/2023-04-21-new-headlamp/index.md

```diff
@@ -31,14 +31,14 @@ The first plugin we are introducing as a part of this is the App Catalog plugin
 
-We have a new [plugins](https://github.com/headlamp-k8s/plugins)  repository where we are maintaining the plugins we officially ship/recommend, and we you can find the App Catalog plugin mentioned above in there. 
+We have a new [plugins](https://github.com/headlamp-k8s/plugins) repository where we are maintaining the plugins we officially ship/recommend, and you can find the App Catalog plugin mentioned above in there. 
 
 As we continue to build out our vision of what a Kubernetes UI should be, expect more plugins to be added to that repository. So, stay tuned for more information on upcoming plugins and features! 
 
 ## Embracing Open Source and Open Development 
 
-The Headlamp team strongly believes in the power of open source software and an open development model. We are convinced that this approach benefits both the project and its users. By fostering collaboration, transparency, and innovation, we can create a vibrant community that will continue to push the boundaries of what a Kubernetes Web UI can achieve. 
+The Headlamp team strongly believes in the power of open source software and an open development model. We are convinced that this approach benefits both the project and its users. By fostering collaboration, transparency, and innovation, we can create a vibrant community that will continue to push the boundaries of what a Kubernetes UI can achieve. 
 
-We hope you join us in this exciting new chapter for the Headlamp project and help us shape the future of Kubernetes Web UIs. We are also looking forward to hearing from you, either on our [Headlamp  channel](https://kubernetes.slack.com/messages/headlamp) in the Kubernetes Slack instance, or on  [Github](https://github.com/headlamp-k8s/headlamp). Whether you're a developer looking to contribute, a vendor looking to build atop Headlamp, or a user seeking a more powerful Kubernetes management experience, we welcome you to be part of this journey. 
+We hope you join us in this exciting new chapter for the Headlamp project and help us shape the future of Kubernetes UIs. We are also looking forward to hearing from you, either on our [Headlamp  channel](https://kubernetes.slack.com/messages/headlamp) in the Kubernetes Slack instance, or on [Github](https://github.com/headlamp-k8s/headlamp). Whether you're a developer looking to contribute, a vendor looking to build atop Headlamp, or a user seeking a more powerful Kubernetes management experience, we welcome you to be part of this journey. 
 
-Together, let's make Kubernetes management more accessible, efficient, and enjoyable for everyone.
\ No newline at end of file
+Together, let's make Kubernetes management more accessible, more efficient, and more enjoyable for everyone.
```

Comment:
I think the first "more" included the efficient and enjoyable. This one reads a bit weird to me, but I'm not a native speaker...

Reply:
it would be normal usage, yes. I repeated the 'more' as a rhetorical device (termed 'anaphora' I believe).

Reply:
I'd prefer to leave the original in this case if that's okay.